src/js/shield_defs.js

```diff
@@ -2397,6 +2397,17 @@ export function loadShields(shieldImages) {
     (county) => (shields[`US:WY:${county}`] = pentagonShieldBlueYellow)
   );
 
+  // SOUTH AMERICA
+
+  // Ecuador
+  shields["EC:national"] = shields["US:I"];
+  shields["EC:provincial"] = {
+    backgroundImage: shieldImages.shield40_ec_collector_3,
+    textLayoutConstraint: ShieldText.southHalfellipseTextConstraint,
+    textColor: Color.shields.white,
+    padding: padding_us_interstate,
+  };
```

Comment:
```suggestion
  shields["EC:provincial"] = {
    ...shields["US:I"],
    backgroundImage: shieldImages.shield40_ec_collector_3,
  };
```

Could probably give business Interstates the same treatment and get rid of `padding_us_interstate` as a separate variable.